crates/anvil-polkadot/src/api_server/server.rs

```diff
@@ -118,14 +120,25 @@ impl ApiServer {
             substrate_service.spawn_handle.clone(),
         )
         .await?;
+
+        // Set up the genesis base_fee_per_gas.
+        let backend = BackendWithOverlay::new(
+            substrate_service.backend.clone(),
+            substrate_service.storage_overrides.clone(),
+        );
+
+        let genesis_hash = backend.blockchain().info().best_hash;
+        backend.inject_next_fee_multiplier(
```

Comment:
Why not do this as part of the GenesisConfig? `base_fee_per_gas` is already on there